src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableRows = table.querySelectorAll('tr');
+const people = [...tableRows].slice(1, -1);
+
+table.firstElementChild.addEventListener('click', (e) => {
```

Comment:
It's better to use event delegation on 'th' elements, rather than on the whole 'thead'. Consider changing the event listener target to 'th' elements.